### DISCUSSION TITLE
Bolder font comparing to other terminal

### DISCUSSION THREAD

**Discussion by lucasrabiec** [wezterm/wezterm#3388] [Q&A] [answered]
Hi, I switched from kitty. I have set MesloLGS Nerd Font in both terminals. In Wezterm lualine plugin (nvim) has the boldest font in some places (like on the screenshot - compared to iterm2 and kitty)

Could someone explain why this difference occurs and how to change it to normal weight?
![CleanShot 2023-03-27 at 21 24 32@2x](https://user-images.githubusercontent.com/9741188/228045974-e532ef1f-e8be-4c10-9731-faa7f0573650.png)

```lua
config = {
  color_scheme = "Dracula (Official)",
  tab_bar_at_bottom = true,
  use_fancy_tab_bar = false,
  font = wezterm.font({ family = "MesloLGS Nerd Font" }),
  font_size = 12,
  window_padding = {
    left = 2,
    right = 2,
    top = 5,
    bottom = 5,
  },
}
```

**Comment by nekowinston** [comment]
You'll want to try to pass in different arguments into the `weight` paremeter:
```lua
wezterm.font({family = "MesloLGS Nerd Font", weight="DemiLight" }))
```

See the docs for [`wezterm.font`](https://wezfurlong.org/wezterm/config/lua/wezterm/font.html)

Also instead of using a Nerd Font with WezTerm, have a look at [`font_with_fallback`](https://wezfurlong.org/wezterm/config/lua/wezterm/font_with_fallback.html)
WezTerm ships with Nerd Font Symbols as a default fallback font, so this shouldn't be needed in most cases.

If you want to add your own fallback font however, you can download [NerdFontsSymbolsOnly.zip](https://github.com/ryanoasis/nerd-fonts/releases/tag/v2.3.3), then specify it explicitly as a fallback font, e.g.:
```lua
wezterm.font_with_fallback({
  { family = "Berkeley Mono", weight = "Bold" },
  "Symbols Nerd Font"
})
```

**Reply by lucasrabiec** [reply]
TY for the answer but I won't change the entire weight because the default (Regular) is ok (in other places). I would like to know why it is different in this specific place (lualine plugin from nvim) in different terminals.

**Reply by nekowinston** [reply]
Does changing [`font_rules`](https://wezfurlong.org/wezterm/config/lua/config/font_rules.html) (specifying a custom weight) help?

**Comment by wez** [comment]
A couple of other things you can try that can affect the appearance of text:

```lua
config.freetype_load_flags = 'NO_HINTING'
```

and/or:

```lua
config.freetype_load_target = 'Light'
config.freetype_render_target = 'HorizontalLcd'
```

and/or:

```lua
config.front_end = "WebGpu"
```

**Comment by lucasrabiec** [comment, marked as answer]
@nekowinston hint helped, TY guys. 
> Does changing [`font_rules`](https://wezfurlong.org/wezterm/config/lua/config/font_rules.html) (specifying a custom weight) help?

I set this and it works.
```
font_rules = {
  {
    intensity = "Bold",
    font = wezterm.font({ family = "MesloLGS Nerd Font", weight = "Regular" }),
  },
},
```

Edit: I changed the font to FiraCode it looks way better and also without changing weight.
